### DISCUSSION TITLE
Fehlermeldung beim beanspruchen der Shimmer Staking Rewards

### DISCUSSION THREAD

**Discussion by Naettili** [iotaledger/firefly#7628] [Help & Support]
Beim beanspruchen der Shimmer Staking Rewards wirdf Wallet1 gefunden, beim der weiteren Suche nach Wallet2 erscheint unten rechts eine Fehlermeldung in Rot," Bei der Suche nach Rewards ist ein Fehler aufgetreten. Dann bleibt nur Wallet1 zu beanspruchen, aber danach besteht in der aktuellen Version keine Möglichkeit Rewards neu zu beanspruchen.
Also Simmer App auf anderem Gerät installieren und neu suchen. Da erscheint dann bei der Suche wieder die Wallet1 als beansprucht und bei weiteren suchen kommt wieder ständig die Fehlermeldung. Hat einer eine Idee für mich, wie ich Wallet2 finden/beanspruchen kann?

**Comment by begonaalvarezd** [comment]
We found a bug and are preparing a release already to fix it, we will release a beta today and a production version very soon after the beta, sorry for the troubles

**Comment by begonaalvarezd** [comment]
Hi, we released a new version fixing this issue  🙏 please upgrade to the new version. We would appreciate if you let if know if the issue persists or not 🌷 thanks!

**Reply by Naettili** [reply]
Hallo Liebes Team,
vielen Dank für die schnelle Info und Hilfe. War jetzt leider viel unterwegs und kann erst heute antworten. Habe das Update installiert und dann wurden die restlichen Shimmer gefunden.
Nochmals vielen Dank für eure Hilfe!
Herzliche Grüße
Naettili
